src/redfish/rest/v1.py

```diff
@@ -267,6 +267,30 @@ def session_location(self):
         self._session_location = self._http_response.getheader('location')
         return self._session_location
 
+    @property
+    def is_processing(self):
+        """Check if we're a PATCH/POST in progress """
+        return self.status == 202
+
+    def process_resource(self, context):
+        """Function to process Task, used on an action or POST/PATCH that returns 202"""
+        my_href = self.getheader('location')
+        if self.is_processing:
+            if my_href:
+                my_content = context.get(my_href, None)

```

Comment:
We might not need to do anything special here; we should be able to just return "context.get(my_href, None)" since the response from the Task Monitor URI will contain everything the caller needs to make a decision about the next step to do.

Reply:
From the discussion today, Location as a header is not needed on subsequent GETs for the Task Monitor itself (not very clear in the spec about this). Would like to see the body of the response copied over to the body of this instance so the tool can get any updated Task info.

Would also be useful to extract the Retry-After header so the tool can use it in their processing loop.